docs/dictionary/property/systemAppearance.lcdoc

```diff
@@ -0,0 +1,30 @@
+Name: systemAppearance
+
+Type: property
+
+Syntax: get the systemAppearance
+
+Summary:
+Determins if the system appearance is dark or light mode. 
+
+Introduced: 9.6
+
+OS: mac, android, ios
+
+Platforms: desktop, mobile
+
+Example:
+if the systemAppearance is "dark" then
+    set the backColor of me to "black"
+    set the foreColor of me to "white"
+else
+    set the backColor of me to "white"
+    set the foreColor of me to "black"
+end if
+
+Description:
+The <systemAppearance> property returns either "dark" or "light" depending on
```

Comment:
I *think* it makes more sense for the default to be 'light' - as discussed offline, it is likely other platforms will get a 'dark' mode following the notion of 'light' for during the day and 'dark' for at night. It means that cross-platform code can use a simple 'if' rather than a switch.